<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YOLOv4 (darknet)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="fotoRadio">
        <property name="text">
         <string>Fotoğraf</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="videoRadio">
        <property name="text">
         <string>Video</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="videoComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="maxVisibleItems">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="videoLinkLine">
      <property name="placeholderText">
       <string>Tanıtılacak video URL'si</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="dosyaYoluLine">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Tanıtılacak dosya yolu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dosyaYoluBtn">
        <property name="text">
         <string>Dosya seç..</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="tanitBtn">
      <property name="text">
       <string>Tanıt!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="darknetLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="darknetConfigAction"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="darknetConfigAction">
   <property name="text">
    <string>darknet Yolunu Belirle</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
